src/v2/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -44,7 +44,7 @@ export class ArtistCollectionsRail extends React.Component<
 
   render() {
     const { collections } = this.props
-    const isMobile = getENV("IS_MOBILE") === true
+    const isMobile = Boolean(getENV("IS_MOBILE")) === true
```

Comment:
@ashleyjelks - the chrome console is handy for testing these sorts of assertions out (also tests :) 

<img width="168" alt="Screen Shot 2020-06-22 at 3 18 22 PM" src="https://user-images.githubusercontent.com/236943/85340822-aaff2580-b49b-11ea-9586-b67bd5fd782a.png">

What you'll want to do is check if something = "true" or "false", as string literals: 
```tsx
getENV("IS_MOBILE") === "true"
```

Reply:
Ah, 😓 , I actually did, test `Boolean("true")` but not `Boolean("false")`!

Reply:
Heh, very familiar with this mistake 😅